bigchaindb/core.py

```diff
@@ -43,6 +43,12 @@ def __init__(self, connection=None):
         """
         config_utils.autoconfigure()
 
+        self.tendermint_host = tendermint_host or bigchaindb.config['tendermint']['host']
+        self.tendermint_port = tendermint_port or bigchaindb.config['tendermint']['port']
+        self.endpoint = 'http://{}:{}/'.format(self.tendermint_host, self.tendermint_port)
+        self.mode_list = ('broadcast_tx_async',
+                          'broadcast_tx_sync',
+                          'broadcast_tx_commit')
```

Comment:
This code has been added to the old `core.py`. As a part of the cleanup, we decided not to add any code to the older files. So, it would be better to move this to `tendermint/lib.py`